### PR TITLE
libdazzle: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

I am disabling Windows for now as it needs to be built with VS2019, which I will do once the feedstock is up and running.

### CODE REVIEW

**Comment by tschoonj** [comment]
@conda-forge/staged-recipes could someone have a look at this recipe? Thanks!

File: recipes/libdazzle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  host:
+    - glib
+    - gtk3
+  run:
+    - gettext  # [osx]
+    - libglib
+    - cairo
+    - pango
+    - gdk-pixbuf
+    - gtk3
```
Comment (xhochy) [inline recipes/libdazzle/meta.yaml:49]: These libraries all have `run_exports` and therefore should only be present in `host`.

Comment (tschoonj) [reply recipes/libdazzle/meta.yaml]: `pango` doesn't seem to have `run_exports`

Comment (xhochy) [reply recipes/libdazzle/meta.yaml]: Oh, it should have one with `max_pin="x"`. We need to fix that.

**Pull request merged**